tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -179,6 +170,13 @@
         // one closest to it, but less than it.
         runToStepCap: Object,
         requestManager: Object,
+        // TODO(chihuahua): Add pagination to the PR curves dashboard, and
```

Comment:
The PR curves dashboard does have pagination:
https://github.com/tensorflow/tensorboard/blob/ab6ffbb552f7a663876dd72a487aded4dff81b0b/tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html#L99-L122

Could you just resolve this TODO now?

Reply:
Yes! Resolved and passed `active` to the `tf-pr-curve-card`.